specification/kubernetesconfiguration/resource-manager/readme.md

```diff
@@ -78,27 +78,26 @@ See configuration in [readme.typescript.md](./readme.typescript.md)
 
 See configuration in [readme.csharp.md](./readme.csharp.md)
 
-## Multi-API/Profile support for AutoRest v3 generators 
+## Multi-API/Profile support for AutoRest v3 generators  
```

Comment:
I do not see meaningful changes in this file. Can you correct the trailing spaces so a clean diff can be reviewed. Thanks!